test/test_tensor_spec.py

```diff
@@ -383,6 +383,302 @@ def test_nested_composite_spec(self, is_complete, device, dtype):
                 assert key in td["nested_cp"].keys()
 
 
+class TestEquality:
+    @staticmethod
+    def _ts_make_all_fields_equal(ts_to, ts_from):
+        ts_to.shape = ts_from.shape
+        ts_to.space = ts_from.space
+        ts_to.device = ts_from.device
+        ts_to.dtype = ts_from.dtype
+        ts_to.domain = ts_from.domain
+        return ts_to
+
+    def test_equality_bounded(self):
+        MIN = 10
```

Comment:
can we rename the caps variables in non caps?
we usually use that for global variables

Reply:
Sure, one moment

Reply:
Done! @vmoens  please check